components/ui/forms/__tests__/input-text.test.tsx

```diff
@@ -1,15 +1,10 @@
-import { render } from "@testing-library/react";
+import { render, screen } from "@testing-library/react";
 import { InputText } from "../input-text";
 
 describe("InputText", () => {
   it("renders correctly", () => {
-    const { container } = render(<InputText />);
+    render(<InputText />);
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <input
-        class="shadow-sm rounded-md focus:ring-blue-500 focus:border-blue-500 block w-full text-sm border-gray-300"
-        type="text"
-      />
-    `);
+    expect(screen.getByRole("textbox")).toBeInTheDocument();
```

Comment:
I think we can keep the .toMatchInlineSnapshot() usage here since we're actually testing the styling for this component and the snapshot result is very small and readable.